tests/CMakeLists.txt

```diff
@@ -1,24 +1,22 @@
-#TODO: we might be able to clean that up a bit and have the include directories propagate with the libs
-include_directories(. googletest/include googletest ../utils ../host_reference)
+# TODO: we might be able to clean that up a bit and have the include directories propagate with the libs
+# include_directories(.  ../utils ../host_reference)
```

Comment:
Delete this?

Reply:
All deleted. Had forgotten to remove that.